src/interfaces/ISecretManagerService.ts

```diff
@@ -23,6 +23,10 @@ export interface TLSCredentials {
   version?: string
 }
 
+export interface CiracConfigSecrets {
```

Comment:
Is the extra 'c' in Cira**c**ConfigSecrets a typo or did you intentionally add it? may be this is just CiraConfigSecrets